staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/matching/matching_test.go

```diff
@@ -536,7 +536,8 @@ func TestMatcher(t *testing.T) {
 
 	for _, testcase := range testcases {
 		t.Run(testcase.name, func(t *testing.T) {
-			matches, matchKind, err := a.Matches(testcase.attrs, interfaces, &fakeCriteria{matchResources: *testcase.criteria})
+			//!TODO: use this var
+			matches, _, matchKind, err := a.Matches(testcase.attrs, interfaces, &fakeCriteria{matchResources: *testcase.criteria})
```

Comment:
TODO

Reply:
added `expectMatchResource` to test cases which use equivalent matching (uses exact match if not supplied), and added a check in test that returned match resource matches expectation